docker-compose.yml

```diff
@@ -113,6 +113,7 @@ services:
           -- celery -A iib.workers.tasks worker --loglevel=info
     environment:
       IIB_DEV: 'true'
+      REGISTRY_AUTH_FILE: '/root/.docker/config.json'
```

Comment:
This should probably be a different commit?

Reply:
@lipoja @pbortlov please let me know what do you think.

For me whatever decision about separating or keeping in a single commit is good

Reply:
Personally, I like separate commits. Each commit should contain only one logical change. 

1. Implementing buildah retries
2. Adding REGISTRY_AUTH_FILE to dev env
3. Updating buildah commands to generate/push manifest v2
4. Static type not related to buildah

Then is we use "rebase and merge" instead of "squash and merge" we end up with these four commits in master branch. Which is good because when something is wrong you can revert the exact commit. 
For example you can buildah revert retries but also keeping commit that is fixing the manifest v2. 

Or if you are not sure if someone will use rebase and merge then you can put all of these to separate PRs. 
My goal is to persuade everybody to do rebase and merge and commit the code properly (one logical change per commit). 

We should also keep in mind if we do it this way that **every commit must work**. 
No commit should break behavior of IIB.

Reply:
Perfect!
Thanks a lot for the clarification, I'll work on it.

Reply:
Push is done, now with separate commits.

Code is the same.